db/changelog/002-create-timeEntry-and-timePeriodType-table.json

```diff
@@ -0,0 +1,18 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "002-create-timeEntry-and-timePeriodType-table",
+        "author": "Callisto Dev",
+        "changes": [
+          {
+            "sqlFile": {
+              "dbms": "postgresql",
+              "path": "sql/create-timeEntry-and-timePeriodType-table.sql"
```

Comment:
Would it be acceptable to include the changelog version number in the sql file? I have visions of the sql folder being full of jumbled up sql files and accidentally looking at the wrong one for a particular change

Reply:
the one level up which is json file is versioned with sequence number

Reply:
I will rename the file to 001-create-timeEntry-and-timePeriodType-table.sql thank you